xml/en/index.xml

```diff
@@ -8,36 +8,51 @@
 <article name="nginx"
          link="/en/"
          lang="en"
-         rev="167">
+         rev="168">
 
 
 <section>
 
 <para>
-nginx [engine x] is an HTTP and reverse proxy server,
-a mail proxy server,
-and a generic TCP/UDP proxy server,
-originally written by <link url="http://sysoev.ru/en/">Igor Sysoev</link>.
-For a long time, it has been running
-on many heavily loaded Russian sites including
-<link url="http://www.yandex.ru">Yandex</link>,
-<link url="http://mail.ru">Mail.Ru</link>,
-<link url="http://vk.com">VK</link>, and
-<link url="http://www.rambler.ru">Rambler</link>.
-Here are some of the success stories:
-<link url="https://blogs.dropbox.com/tech/2017/09/optimizing-web-servers-for-high-throughput-and-low-latency/">Dropbox</link>,
-<link url="https://openconnect.netflix.com/en/software/">Netflix</link>,
-<link url="http://blog.fastmail.fm/2007/01/04/webimappop-frontend-proxies-changed-to-nginx/">FastMail.FM</link>.
+nginx ("<i>engine x</i>") is an HTTP web server, reverse proxy,
+content cache, load balancer,
+TCP/UDP proxy server,
+and mail proxy server.
+Originally written by <link url="http://sysoev.ru/en/">Igor Sysoev</link>,
```

Comment:
comma looks suspicious, please break in two sentences

Reply:
How about this:

> NGINX was originally written by [Igor Sysoev](http://sysoev.ru/en/) and is distributed under the terms of a simplified [2-clause BSD-like license](https://github.com/nginx/nginx/blob/update-readme/LICENSE).

I don't think we need to explain (in this paragraph) that the documentation has the same license.

Reply:
I'm fine about skipping documentation license part.
I'm not fine about "simplified": simplified means exactly the same as 2-clause BSD license.
Probably also skipping "-like" part because nginx license text exactly follows 2-clause BSD license text.
https://opensource.org/license/bsd-2-clause